jac-uk/admin
# make pagination numerical by default

## What's included?
Alphabetical pagination on the stat-con report page was causing confusion with the stat-con download. 
I have made the pagination numerical by default so this page should always load any results, reducing confusion. 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
visit stat con page (exercise >> reports >> statutory-consultation)
ensure pagination is numerical on initial load.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
